prime-router/docker-compose.build.yml

```diff
@@ -0,0 +1,45 @@
+# Build environment
+version: "3.3"
+services:
+  builder:
+    depends_on:
+      - postgresql
+    build:
+      context: .
+      dockerfile: Dockerfile.build
+    environment:
+      FUNCTIONS_CORE_TOOLS_TELEMETRY_OPTOUT: 1
+      DB_URL: jdbc:postgresql://postgresql:5432/prime_data_hub
+    volumes: # Attach the pwd into the image
+      - type: bind
+        source: ./
+        target: /src
```

Comment:
Nice! So any artifacts built in the container will be available outside the container.

Reply:
Exactly, and this is what makes it so that you can first build through the container, then just invoke `gradle[w]`, then through the container again, etc... make it as frictionles as possible :)